Taarafo.Core/Migrations/20211110140549_UpdateCommentModel.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Taarafo.Core.Migrations
+{
+    public partial class UpdateCommentModel : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<DateTimeOffset>(
+                name: "CreatedDate",
+                table: "Comments",
+                type: "datetimeoffset",
+                nullable: false,
+                defaultValue: new DateTimeOffset(new DateTime(1, 1, 1, 0, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, 0, 0, 0, 0)));
+
+            migrationBuilder.AddColumn<DateTimeOffset>(
+                name: "UpdatedDate",
+                table: "Comments",
```

Comment:
These changes don't seem to be related to a controller task

Reply:
@hassanhabib I had to add this to get the controller to return a valid output as the migrations was not up to date.

Reply:
@cjdutoit let's have a different pr for this then. `DATA: Add Created & Updated Dates`

Reply:
Submitted PR #78 to address this.

Reply:
@cjdutoit perfect. thanks. I just left you one tiny comment there.

Reply:
closed via #78